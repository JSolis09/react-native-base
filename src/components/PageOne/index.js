import React, {Component} from 'react';
import { StyleSheet, Text, View} from 'react-native';
import { COLOR } from 'react-native-material-ui';

const styles = StyleSheet.create({
    container: {
      flex: 1,
      justifyContent: 'center',
      alignItems: 'center',
      backgroundColor: COLOR.grey100,
    },
    content: {
        fontSize: 20,
        textAlign: 'center',
        margin: 10,
    },
});

export default class PageOne extends Component {
    render() {
        return(
            <View style={styles.container}>
                <Text style={styles.content}>
                    Page One
                </Text>
            </View>
        )
    }
}