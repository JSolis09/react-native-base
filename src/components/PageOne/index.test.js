import React from 'react';
import { create } from 'react-test-renderer';
import PageOne from './index';

describe('Home Component', () => {
    it('should render correctly', () => {
        const tree = create(<PageOne />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});