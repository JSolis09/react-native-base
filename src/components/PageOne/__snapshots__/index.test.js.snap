// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#f5f5f5",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Page One
  </Text>
</View>
`;
